sl/CMakeLists.txt

```diff
@@ -16,12 +16,7 @@
 # along with predator.  If not, see <http://www.gnu.org/licenses/>.
 
 # project metadata
-if(APPLE)
-    # because of MACOSX_RPATH
-    cmake_minimum_required(VERSION 3.0)
```

Comment:
This condition was originally added because of `MACOSX_RPATH`.  Now you are removing it without any explanation.  I can see that `build-aux/common.cmake` still uses `MACOSX_RPATH`.  Could you please clarify it?

Reply:
`get_property` ends with error => unsupported for version 3.0+. I mentioned it before.

Reply:
Honestly, I do not understand which problem the `MACOSX_RPATH` macro solves and how it works, neither how it relates to `get_property` and/or CMake 3.0+.  But, given the fact that the proposed change seems to be a no-op for anything but Apple, I am fine with it.